content/posts/ytnapariparty.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "A yt-napari party"
+date: 2021-01-12T18:22:25-06:00
+author: Chris Havlin
+cover: /img/ytnapari/ytnapari_enzo3d.png
+categories:
+  - Tutorial
+tags:
+  - Community
```

Comment:
```suggestion
  - Experimental
```

I think this should have a new tag. Community and development are more about governance and social things with the project. What do you think?

Reply:
yes!! That's what I needed. `Experimental` is great!